indico/modules/vc/models/vc_rooms.py

```diff
@@ -351,6 +351,21 @@ def delete(self, user):
             db.session.delete(self.vc_room)
 
 
+    def delete_all(self, user):
```

Comment:
Why not add a `force_delete_room` arg that defaults to `False` to `delete()` to delete the room even if it's still linked to other events?